cmd/serverless-init/main.go

```diff
@@ -235,3 +240,14 @@ func setEnvWithoutOverride(envToSet map[string]string) {
 		}
 	}
 }
+
+func errorExitCode(err error) int {
+	// if error is of type exec.ExitError then propagate the exit code
+	var exitError *exec.ExitError
+	if errors.As(err, &exitError) {
+		return exitError.ExitCode()
+	}
```

Comment:
Is it useful to add a log here, in case the exit code cannot be decoded?
It could remove the ambiguity between returning 1 from our code and exitcode 1

Reply:
Done! I also added a `log.Flush()` before exiting like we discussed so these logs are reliability generated.